cleanlab/latent_algebra.py

```diff
@@ -33,21 +33,21 @@
 
 
 def compute_ps_py_inv_noise_matrix(s, noise_matrix):
-    """Compute ps := P(s=k), py := P(y=k), and the inverse noise matrix.
+    """Compute ps := P(labels=k), py := P(y=k), and the inverse noise matrix.
 
     Parameters
     ----------
 
     s : np.array
-        A discrete vector of labels, s, which may contain mislabeling. "s"
+        A discrete vector of labels, labels, which may contain mislabeling. "labels"
         denotes the noisy label instead of \\tilde(y), for ASCII reasons.
```

Comment:
Style the second labels differently. Also, the second statement is now obsolete since we're no longer using 's'.

Reply:
(i think) i've gotten this fixed everwhere. I left ASCII note in one place in the code, but removed everywhere else.